typescript/pnpm-p5js/.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
not sure we want to include .envrc into the templates

Reply:
The rust template has a .envrc, do I remove it from here? Or should I just do like a pull request removing it elsewhere as well?